.github/workflows/e2e.yml

```
@@ -21,24 +21,10 @@ jobs:
-        options: >-
-          --health-cmd "mongosh --quiet --eval 'db.runCommand({ ping: 1 }).ok' | grep -q 1"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 15
-
-    env:
-      # Server configuration — these are CI-only, not production secrets
-      MONGO_URL: mongodb://localhost:27017
-      DB_NAME: llm_relay_e2e
+STORAGE_BACKEND: sqlite
       KEYS_FILE: e2e-keys.json
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)